File: Go scripts/check_website/check_website.go
package main

import (
	"fmt"
	"net/http"
	"time"
)

func main() {
	url := "https://www.netacad.com/"

	client := http.Client{
		Timeout: 5 * time.Second,
	}

	resp, err := client.Get(url)
	if err != nil {
		fmt.Printf("[ERROR] Cannot reach %s: %v\n", url, err)
		return
	}
	defer resp.Body.Close()

	if resp.StatusCode == 200 {
		fmt.Printf("[OK] %s is reachable!\n", url)
	} else {
		fmt.Printf("[ERROR] %s returned status code %d\n", url, resp.StatusCode)
	}
}

File: Python scripts/check_website.py
import requests

# Website to check
url = "https://azure.microsoft.com/en-us/"

try:
    response = requests.get(url, timeout=5)  # timeout in seconds
    if response.status_code == 200:
        print(f"[OK] {url} is reachable!")
    else:
        print(f"[ERROR] {url} returned status code {response.status_code}")
except requests.exceptions.RequestException as e:
    print(f"[ERROR] Cannot reach {url}: {e}")
